components/atoms/CopyToClipboard/CopyToClipboardStyles.ts

```diff
@@ -0,0 +1,26 @@
+import ContentCopyIcon from 'mdi-react/ContentCopyIcon'
+import styled from 'styled-components'
+
+export const StyledCopyMessage = styled.span`
+    color: #a112ff;
+`
+export const StyledCopyContainer = styled.div`
+    align-items: center;
+    display: flex;
+    position: absolute;
+    top: 4px;
+    right: 0;
+`
+export const StyledCopyButtonContainer = styled.div`
+    position: relative;
+`
+export const StyledCopyButton = styled.button`
+    border-color: transparent;
+    background-color: transparent;
+    cursor: pointer;
+`
+export const StyledCopyIcon = styled(ContentCopyIcon)`
+    :hover {
+        color: #a112ff;
```

Comment:
Maybe this should be blurple?

Reply:
Yeah, I think blurple / 5033e1 is nicer and would offer more contrast, but I would still run it through an accessibility check

Reply:
Went with one shade up from blurple, which passed

Reply:
Cool, thanks